src/main/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannel.java

```diff
@@ -128,7 +135,10 @@ public void insertRecordToBuffer(SinkRecord record) {
 
       // broken record
       if (isRecordBroken(snowflakeRecord)) {
-        // write it to DLQ SNOW-451197
+        // check for error tolerance and log tolerance values
+        // errors.log.enable and errors.tolerance
+        LOGGER.debug("Broken record offset:{}, topic:{}", record.kafkaOffset(), record.topic());
+        this.kafkaRecordErrorReporter.reportError(record, new DataException("Broken Record"));
```

Comment:
I assume there will follow-up PRs to send records to DLQ when insertRows API failed?

Reply:
yes..